src/style.css

```diff
@@ -0,0 +1,3 @@
+body {
+  margin: 0;
+}
```

Comment:
Write your styles using classes.
Move styles from style.css to another .scss file.